bndtools.m2e.runtime/bnd.bnd

```diff
@@ -0,0 +1,7 @@
+-resourceonly: true
+
+Bundle-SymbolicName: bndtools.m2e.runtime; singleton:=true
```

Comment:
So this is not part of the shipped feature? OK, then perhaps we should rename this project to be more explicit that it is only for local test/debug.